src/System.Threading.Overlapped/src/System.Threading.Overlapped.builds

```diff
@@ -2,9 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Threading.Overlapped.CoreCLR.csproj">
```

Comment:
Reminder: Once this merges internally this will break the TFS build so please watch this and be ready with a fix internally once this mirrors.